lang/julia/Portfile

```diff
@@ -86,7 +86,6 @@ post-destroot {
     move ${dpw}/julia-${version}/include/julia ${destroot}${prefix}/include
     move ${dpw}/julia-${version}/lib/julia ${destroot}${prefix}/lib
     move ${dpw}/julia-${version}/lib/libjulia.${short_version}.dylib ${destroot}${prefix}/lib
-    move ${dpw}/julia-${version}/lib/libjulia.${short_version}.dylib.dSYM ${destroot}${prefix}/lib
```

Comment:
I didn't write this original code and am only vaguely aware why it's there. It's been robust for a long time now.

If we're going to edit it, would you please document why in the PR discussion for posterity?

Also, a change like this may require incrementing the `revision` if any installed library or binary files changed.